### PR TITLE
RSDK-2269 - flaky test: services/motion/builtin TestMoveSingleComponent

### PR DESCRIPTION
**Pull request by nfranczak**
re- https://viam.atlassian.net/browse/RSDK-2269

doing:
```
go test -run TestMoveSingle/succeeds_with -count=100
PASS
ok      go.viam.com/rdk/services/motion/builtin 129.642s
```
average time 1.29642s

### CODE REVIEW

File: services/motion/builtin/builtin_test.go
Comment (nfranczak) [inline services/motion/builtin/builtin_test.go:190]: new inputs are modeled after:
https://github.com/viamrobotics/rdk/blob/main/components/arm/arm_test.go#L455

**Comment by nfranczak** [comment]
this is from the CI logs:
```
--- PASS: TestMoveSingleComponent (16.38s)
    --- PASS: TestMoveSingleComponent/succeeds_when_all_frame_info_in_config (4.89s)
    --- PASS: TestMoveSingleComponent/fails_due_to_gripper_not_being_an_arm (0.00s)
    --- PASS: TestMoveSingleComponent/succeeds_with_supplemental_info_in_world_state (11.44s)
```

unsure why the CI tests for succeeds_with_supplemental_info_in_world_state took roughly 10x longer.

**Review by Otterverse — APPROVED** [review]
I don't know this area of code enough to really comment in detail, but If it fixes the flaky test, then LGTM.

**Comment by nfranczak** [comment]
@Otterverse I will wait for approval and confirmation that this fixes the flaky test from @spieswl & @raybjork before merging.

File: services/motion/builtin/builtin_test.go
Comment (nfranczak) [reply services/motion/builtin/builtin_test.go:190]: our home position is:
```
startPose x:-817.2 y:-232.9 z:62.8 o_x:1.5700924586837752e-16 o_y:-1 o_z:1.7938038903913487e-16 theta:89.99999999999999
```

since our goal is wrt another frame, its converted point is:
```
builtin/builtin.go:165  converted goal pose "x:-796.2 y:-231.9 z:63.8 o_z:1"
```

**Review by raybjork — APPROVED** [review]

**Comment by nfranczak** [comment]
to try and convince myself that this solves the flake, I read through some of the new logger statements.
our home position is:
```
startPose x:-817.2 y:-232.9 z:62.8 o_x:1.5700924586837752e-16 o_y:-1 o_z:1.7938038903913487e-16 theta:89.99999999999999
```
since our goal is wrt another frame, its converted point is:
```
builtin/builtin.go:165  converted goal pose "x:-796.2 y:-231.9 z:63.8 o_z:1"
```
this converted point seems like it should be easier to solve for…
the pose which was causing the flake had
```
converted goal pose "x:60.999999999999915 y:-230 z:793 o_x:4.655546057013451e-16 o_y:-0.7071067811865476 o_z:-0.7071067811865475 theta:-44.99999999999996"
```
which seems much more difficult to solve for.

**Review by spieswl — APPROVED** [review]
Good follow-up, LGTM.

**Pull request merged**
